util/Evaluate.py

```diff
@@ -20,10 +20,12 @@
 def digitize(tensor, bin_edges, device, middle='true', dtype=torch.float32):
     bin_edges = torch.tensor(bin_edges, device=torch.device(device))
     bin_indices = torch.bucketize(tensor, bin_edges) - 1
+    Mask = (bin_indices >= len(bin_edges) - 1) | (bin_indices == 1)
```

Comment:
Am I correct in my understanding that this is to mask out hits that fall into the last bin i.e. preventing it from displaying the 'overflow'?

Reply:
Yes, and also the first bin i.e. before the detector showering.